models/articles.model.js

```diff
@@ -1,6 +1,18 @@
 const { promises } = require("supertest/lib/test.js")
 const db = require("../db/connection.js")
 
+
+exports.fetchArticles = (properties = '*', sortBy = 'created_at', orderBy = 'DESC') => {
```

Comment:
properties isn't required here, but you've correctly put the default parameters for sortBy and orderBy here! ⭐